pkg/apis/core/validation/validation_test.go

```diff
@@ -20134,3 +20136,26 @@ func TestValidateOS(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateAppArmorProfileFormat(t *testing.T) {
+	tests := []struct {
+		profile     string
+		expectValid bool
+	}{
+		{"", true},
+		{v1.AppArmorBetaProfileRuntimeDefault, true},
+		{v1.AppArmorBetaProfileNameUnconfined, true},
+		{"baz", false}, // Missing local prefix.
+		{v1.AppArmorBetaProfileNamePrefix + "/usr/sbin/ntpd", true},
+		{v1.AppArmorBetaProfileNamePrefix + "foo-bar", true},
```

Comment:
Let's add a test for `{v1.AppArmorBetaProfileNamePrefix, false},`

Reply:
That doesn't work - it's addressed by the comment in `apparmor/validate.go`:
```
// TODO(#64841): This would ideally be part of validation.ValidateAppArmorProfileFormat, but
// that is called for API validation, and this is tightening validation.
```